README.md

```diff
@@ -39,9 +39,14 @@ them. Check out the documentation [on docs.rs][docs] (or on
 ## Status
 
 As of 2021-02-10, we support all events and REST endpoints the latest released
-versions of the various Matrix APIs. Various changes from the unstable version
-of the specifications and some MSCs are also implemented, gated behind the
-`unstable-pre-spec` Cargo feature.
+versions of the various Matrix APIs.
```

Comment:
If you're touching this, can you update it to the current date and change it to "v1 of the matrix specification" or similar instead of "the latest released"?